### PR TITLE
Frontend + appservice

### PR DESCRIPTION
**Pull request by Justinluu2311**
Have split the frontend and appservice

### CODE REVIEW

File: team14-app/.gitignore [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# See https://docs.github.com/get-started/getting-started-with-git/ignoring-files for more about ignoring files.
```
Comment (dtronmans) [inline team14-app/.gitignore]: Why is there a team14-app/.gitignore file and a .gitignore file in the root directory? Everything should be in the root directory.

Comment (dtronmans) [reply team14-app/.gitignore]: Same for the Readme

File: README.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
 # app
 URL fishing detection application containing frontend and service
+
+## Model service
+
+To configure the URL of the model service, make edit the environment.ts file and set modelServiceURL="YOUR_URL"
```
Comment (n4dinuh) [inline README.md]: Can you use backticks for better readability? e.g. `environments.ts`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,6 @@
 # app
```
Comment (n4dinuh) [inline README.md]: Could you extend this README with more information? For example:
- What is the purpose of this model service
- What is the main entrypoint and how to start up the app
- What are its main components (and possibly how they connect to each other)

Comment (n4dinuh) [reply README.md]: Ok I see there is another README with instructions - can you merge these 2 into 1 and add the missing points above? ;)

**Review by n4dinuh — CHANGES_REQUESTED** [review]

File: team14-app/.gitignore (hunk 1 above)
Comment (Justinluu2311) [reply team14-app/.gitignore]: Ok, changed it. Please check again.

File: README.md (hunk 2 above)
Comment (Justinluu2311) [reply README.md]: Ok, changed it. Please check again.

File: README.md (hunk 3 above)
Comment (Justinluu2311) [reply README.md]: Ok, changed it. Please check again.

**Review by n4dinuh — APPROVED** [review]
README is looking much better now!

**Comment by 07Liyang** [comment]
great job！

**Pull request closed without merging**